docs/batfish.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const {
   buildNavigation,
-  buildTopics,
+  buildItems,
```

Comment:
I generalized the `buildTopics` function to accept a new argument denoting which "filterable" to generate data for.